ops/model.py

```diff
@@ -1244,15 +1250,18 @@ class Relation:
             For subordinate relations, this set will include only one unit: the principal unit.
         data: A :class:`RelationData` holding the data buckets for each entity
             of a relation. Accessed via eg Relation.data[unit]['foo']
+        active: Indicates whether this relation is active (True) or inactive (False, because
+            this relation is currently part of a relation-broken event)
     """
 
     def __init__(
             self, relation_name: str, relation_id: int, is_peer: bool, our_unit: Unit,
-            backend: '_ModelBackend', cache: '_ModelCache'):
+            backend: '_ModelBackend', cache: '_ModelCache', active: bool = True):
```

Comment:
For this one, I'd probably use a required (non-keyword) argument -- that is, don't default to `True`, but require `active` to be provided. My rationale is that we should not forget to specify `active` correctly on all call sites.

That would have caught the fact that we've forgotten here to specify `active=False` in the other call to `Relation()` in `_get_unique`. See the comment "The relation may be dead, but it is not forgotten." I believe that case should always set active to False.